### ISSUE TITLE
Background

### ISSUE THREAD

**Issue by Default4152** [andrew-lof/bwa-u4-feature-request#1]
I like the idea and you have good reasons to why this should be implemented, especially the detail about accessibility. I also appreciated the link to the UX discussion detailing why it's a good idea.
